kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -36,7 +37,9 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
   val credentialsManager = new HadoopCredentialsManager()
 
   override def initialize(conf: KyuubiConf): Unit = {
-    addService(credentialsManager)
+    if (UserGroupInformation.isSecurityEnabled) {
+      addService(credentialsManager)
+    }
```

Comment:
cc @zhouyifan279

Reply:
This `if` statement should be placed in `HadoopCredentialsManager#initialize` like how we do in `KinitAuxiliaryService#initialize`. 
Otherwise, we will use an uninitialized `HadoopCredentialsManager`.